### PR TITLE
Extends operators

### PR DESCRIPTION
**Pull request by privat**
* Add missing combined assignments: `*=`, `/=`, `%=`, `**=`, `<<=`, `>>=`
* Add missing unary + operator (why?)
* Keep operators in the AST for better messages
* Factorize some information in the AST to simplify code in other modules

### CODE REVIEW

File: src/parser/parser.nit [hunk 1]
```diff
@@ -6344,6 +6354,96 @@ private class ReduceAction241
...
+					)
+					node_list = passignopnode1
+					p.push(p.go_to(_goto), node_list)
+	end
+end
+private class ReduceAction242
+	super ReduceAction
+	redef fun action(p: Parser)
+	do
+					var node_list: nullable Object = null
```
Comment (Morriar) [inline src/parser/parser.nit]: First time I remark that the generated indent is weird

**Comment by Morriar** [comment]
+1

**Comment by R4PaSs** [comment]
:shipit:

**Comment by privat** [comment]
It's rare that @Morriar and @R4PaSs ack a PR without mentioning spelling or typos.

**Comment by R4PaSs** [comment]
I guess that's a call to...

**Pull request merged**